ciri/BUILD

```diff
@@ -1,10 +1,26 @@
 package(default_visibility = ["//visibility:public"])
 
+load(":conf.bzl", "CIRI_MAINNET_VARIABLES")
+load(":conf.bzl", "CIRI_TESTNET_VARIABLES")
+
+config_setting(
+    name = "mainnet",
+    values = {"define": "network=mainnet"},
+)
+
+config_setting(
+    name = "testnet",
+    values = {"define": "network=testnet"},
+)
```

Comment:
Do not create new  `config_setting` entries for this.
Instead, we should move the generic `config_setting` flags to, e.g., `//config/BUILD` and then reuse them here.
(e.g. right now, we already have `//gossip:mainnet` and `//gossip:testnet` that describe this parameter)

Reply:
`consensus` also has same `config_setting` entries. Should I modify them all together?

Reply:
That'd be awesome :)